sections/exchange/TradeCard/Cards/MarketDetailsCard.tsx

```diff
@@ -92,8 +101,7 @@ const MarketDetailsCard: FC<MarketDetailsCardProps> = ({ currencyKey, priceRate,
 				{rates24High != null
 					? `${formatCurrency(selectedPriceCurrency.name, rates24High, {
 							sign: selectedPriceCurrency.sign,
-							minDecimals:
-								currencyKey === SYNTHS_MAP.sKRW || currencyKey === SYNTHS_MAP.sJPY ? 4 : 2,
+							minDecimals: /*currencyKey === Synths.sKRW || */ currencyKey === Synths.sJPY ? 4 : 2,
```

Comment:
why get rid of sKRW?

Reply:
`Synths` does not define sKRW (but it should). I've added it back manually but we need to correct the `Synths` enum to match product.

Reply:
+1